v3/generator/client/client.go

```diff
@@ -39,23 +40,17 @@ func Generate(doc libopenapi.Document, path, packageName string) error {
 	)
 	`, packageName))
 
-	for _, s := range r.Model.Servers {
-		api, err := extractAPIName(s)
-		if err != nil {
-			return err
-		}
-
-		if api != "api" {
-			// Skip generating code for pre-production "ppapi" server.
-			continue
-		}
+	// The spec is returning only production server.
+	if len(r.Model.Servers) != 1 {
```

Comment:
I feel we should not stop execution if more then one server is found.
It could be just a warn and we continue with the first value, just for the sake of distinction between blocker and non-blocker type of errors.

Reply:
https://github.com/exoscale/egoscale/pull/631/commits/4f0f2f73d82c375e5ae4f607d5841dd97bc4270a